lib/index.js

```diff
@@ -46,15 +46,14 @@ module.exports = {
     'consistent-this': [2, 'self'],
     'new-parens': 2,
     'no-array-constructor': 2,
-    'no-arrow-condition': 2,
+    'no-confusing-arrow': 2,
     'no-new-object': 2,
     'no-spaced-func': 2,
     'no-mixed-spaces-and-tabs': 2,
-    'space-after-keywords': 2,
+    'keyword-spacing': 2,
```

Comment:
:+1:

I had that change, but wasn't sure if that was best (because it enforces never allowing spaces). I'll make that change and fix the brace style test